src/app/shared/components/template/components/audio/audio.component.ts

```diff
@@ -9,7 +9,39 @@ import { Howl } from "howler";
 import { IonRange } from "@ionic/angular";
 import { ITemplateRowProps } from "../../models";
 import { TemplateBaseComponent } from "../base";
-import { TemplateAssetService } from "../../services/template-asset.service";
+import { PLHAssetPipe } from "../../pipes/plh-asset.pipe";
+
+// Names of ion-icons to be used by default in the player.
+// Will be overridden if user provides values for play_icon_asset, pause_icon_asset or forward_icon_asset
+const PLAY_ICON_DEFAULT = "play-outline";
+const PAUSE_ICON_DEFAULT = "pause-outline";
+const FORWARD_ICON_DEFAULT = "play-forward";
+
+interface IAudioParams {
+  /** TEMPLATE PARAMETER: "src". Will be overridden by a value passed as "value" to the component. */
+  src: string;
+  /** TEMPLATE PARAMETER: "play_icon_asset". The path to an svg to override the default play icon. Default icon is ion's "play-outline" */
+  playIcon: string;
```

Comment:
nit(non-blocking)
would it make life any simpler keeping template parameter names and local param names the second? I know we would lose a bit in naming convention, but probably less prone to typos that way?
But otherwise definitely like how you've kept all the template params together in a single interface

Reply:
Sounds good to me, yeah happy to just try be consistent then with author `snake_case`, code `camelCase` and using same words.